Yafc/Workspace/ProductionTable/ProductionTableView.cs

```diff
@@ -120,7 +121,22 @@ private static void BuildRowMarker(ImGui gui, RecipeRow row) {
     private class RecipeColumn(ProductionTableView view) : ProductionTableDataColumn(view, LSs.ProductionTableHeaderRecipe, 13f, 13f, 30f, widthStorage: nameof(Preferences.recipeColumnWidth)) {
         public override void BuildElement(ImGui gui, RecipeRow recipe) {
             gui.spacing = 0.5f;
-            switch (gui.BuildFactorioObjectButton(recipe.recipe, ButtonDisplayStyle.ProductionTableUnscaled)) {
+            var buttonEvent = gui.BuildFactorioObjectButton(recipe.recipe, ButtonDisplayStyle.ProductionTableUnscaled);
+
+            // Track hovered recipe for Ctrl+C functionality and show tooltip
+            if (gui.IsMouseOver(gui.lastRect)) {
+                view.hoveredRecipe = recipe;
+
+                // Show tooltip with Ctrl+C hint if the recipe has an entity (can generate blueprint)
+                if (recipe.entity != null) {
+                    gui.ShowTooltip(gui.lastRect, tooltip => {
+                        tooltip.BuildText(recipe.recipe.target.locName, Font.subheader);
+                        tooltip.BuildText("Press Ctrl+C to copy building blueprint", TextBlockDisplayStyle.WrappedText);
```

Comment:
Since #475 we have i18n, so I guess this needs to change a bit. I never used it, so I guess @DaleStan knows better.

Reply:
I'm not too concerned about this, since no one has submitted translations. But yes, the correct way to add new UI strings to Yafc is to add them to Yafc/Data/locale/en/yafc.cfg, and then pass something like `LSs.ProductionTableCopyBuildingBlueprint` (depending on the localization key used in yafc.cfg) instead of the literal string.